pytket/binders/zx/diagram.cpp

```diff
@@ -404,6 +404,11 @@ void ZXDiagramPybind::init_zxdiagram(py::module& m) {
           "remove_wire",
           (void(ZXDiagram::*)(const Wire&)) & ZXDiagram::remove_wire,
           "Removes the given wire from the diagram.", py::arg("w"))
+      .def(
+          "to_circuit", &zx_to_circuit,
```

Comment:
It should be reasonably predictable for this direction, since the diagram must be unitary (hence only contains quantum boundaries) and the ith input/output should get mapped to `Qubit(i)` in the circuit, but it never hurts to provide the map explicitly. However, I just spotted that the map in conversion from circuits to ZX will only contain an entry for either the input or output of each given unit, rather than both, since it is just a map from a unitid to a single vertex. I will modify this alongside adding a map here.